makeflow/src/mf_mesos_executor

```diff
@@ -0,0 +1,137 @@
+import os
```

Comment:
This would be a good place to have a long explanatory comment about the makeflow-mesos architecture, who this component works, and why it is necessary.